README.md

```diff
@@ -233,10 +233,10 @@ nebulous. While nebulous, it guides this crate's architecture and the sorts of
 the trade offs it makes. For example, here are some general architectural
 statements that follow as a result of the goal to be "fast":
 
-* When given the choice between faster regex searches and faster Rust compile
-times, this crate will generally choose faster regex searches.
-* When given the choice between faster regex searches and faster regex compile
-times, this crate will generally choose faster regex searches. That is, it is
+* When given the choice between faster regex searches and faster _*Rust compile
+times*_, this crate will generally choose faster regex searches.
+* When given the choice between faster regex searches and faster _*regex compile
+times*_, this crate will generally choose faster regex searches. That is, it is
```

Comment:
Please wrap to 79 columns (inclusive).

Reply:
I just removed the double emphasis and just italicized the differences to keep the current wrapping.

Reply:
italicize only, and removed bold styling to keep current line wrapping to 79 cols inclusive.